docs/autogen/apigen.py

```diff
@@ -156,19 +156,25 @@ def main(args):
     logging.info("Output: %r", args.output)
 
     namespaces = [
-        "xnvme_adm",
+        "xnvme_buf",
+        "xnvme_mem",
+        "xnvme_cli",
         "xnvme_dev",
         "xnvme_file",
         "xnvme_geo",
         "xnvme_ident",
-        "xnvme_libconf",
+        "xnvme_cmd",
+        "xnvme_adm",
+        "xnvme_kvs",
         "xnvme_nvm",
+        "xnvme_znd",
+        "xnvme_lba",
+        "xnvme_queue",
+        "xnvme_libconf",
         "xnvme_opts",
         "xnvme_spec",
         "xnvme_util",
         "xnvme_ver",
-        "xnvme_znd",
-        "xnvmec",
```

Comment:
What's going on here? 
Maybe they should be alphabetical?

Reply:
Yeah, sorted now. As I recall, then there used to be a dependency on the ordering of these for the api-generator.